activestorage/app/models/active_storage/blob/representable.rb

```diff
@@ -28,7 +28,7 @@ module ActiveStorage::Blob::Representable
   # This will create a URL for that specific blob with that specific variant, which the ActiveStorage::RepresentationsController
   # can then produce on-demand.
   #
-  # Raises ActiveStorage::InvariableError if ImageMagick cannot transform the blob. To determine whether a blob is
+  # Raises ActiveStorage::InvariableError if the variant processor cannot transform the blob. To determine whether a blob is
```

Comment:
Can you wrap this line to 80 characters? We try to wrap doc changes to 80 characters.
https://github.com/rails/rails/pull/40524#discussion_r516677441